src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -338,9 +336,24 @@ private void startTestItem(ExtensionContext context, List<Object> arguments, Ite
 		} else {
 			String codeRef = getCodeRef(context, "");
 			rq.setCodeRef(codeRef);
-			Method testMethod = getTestMethod(context);
-			rq.setAttributes(getAttributes(testMethod));
-			TestCaseIdEntry caseId = getTestCaseId(testMethod, codeRef, arguments);
+			Optional<Method> testMethod = getTestMethod(context);
+			TestCaseIdEntry caseId = testMethod.map(m -> {
+				rq.setAttributes(getAttributes(m));
+
+				Parameter[] params = m.getParameters();
+				rq.setParameters(IntStream.range(0, arguments.size()).boxed().map(i -> {
```

Comment:
Logic of params parsing can be moved to the separate method

Reply:
done